src/index.js

```diff
@@ -0,0 +1,39 @@
+import './style.css';
+import Tasks from './functionalities.js';
+
+const tasksList = new Tasks();
+
```

Comment:
These spaces have made your code very neat but when over used they can make your code confusing and unnecessarily lengthy and tedious.